payjoin-defaults/src/lib.rs

```diff
@@ -13,20 +11,49 @@ use payjoin::Url;
 /// parameter is only available when the "danger-local-https" feature is enabled.
 #[cfg(feature = "v2")]
 pub fn fetch_ohttp_keys(
-    ohttp_relay: Url,
-    payjoin_directory: Url,
-    #[cfg(feature = "danger-local-https")] cert_der: Vec<u8>,
+    ohttp_relay: payjoin::Url,
+    payjoin_directory: payjoin::Url,
+) -> Result<payjoin::OhttpKeys, Error> {
+    let ohttp_keys_url = payjoin_directory.join("/ohttp-keys")?;
+    let proxy = reqwest::Proxy::all(ohttp_relay.as_str())?;
+    let client = reqwest::blocking::Client::builder().proxy(proxy).build()?;
+    let mut res = client.get(ohttp_keys_url).send()?;
+    let body = {
+        let mut body = Vec::new();
+        let _ = res.copy_to(&mut body)?;
+        body
+    };
+    payjoin::OhttpKeys::decode(body.as_slice())
+        .map_err(|e| Error(InternalError::InvalidOhttpKeys(e.to_string())))
+}
```

Comment:
This can be simplified quite a bit

```rs
pub async fn fetch_ohttp_keys(
    ohttp_relay: payjoin::Url,
    payjoin_directory: payjoin::Url,
) -> Result<payjoin::OhttpKeys, Error> {
    let ohttp_keys_url = payjoin_directory.join("/ohttp-keys")?;
    let proxy = reqwest::Proxy::all(ohttp_relay.as_str())?;
    let client = reqwest::Client::builder().proxy(proxy).build()?;
    let res = client.get(ohttp_keys_url).send().await?;
    let body = res.bytes().await?.to_vec();
    payjoin::OhttpKeys::decode(&body)
        .map_err(|e| Error(InternalError::InvalidOhttpKeys(e.to_string())))
}
```